Error: Navigation lists must be nested in a `nav` element
Error: Current page must be indicated using `aria-current="page"`
Error: Styles were not requested
Error: Style changes (such as bold) must not be the only means of indicating a link is the current page
diff --git a/output/Basic Navigation with Current Page/3/f8a4851c-2e2c-4078-93db-1edbf8c2d05b.html b/output/Basic Navigation with Current Page/3/f8a4851c-2e2c-4078-93db-1edbf8c2d05b.html
index 30b2f0c..50e0a3d 100644
--- a/output/Basic Navigation with Current Page/3/f8a4851c-2e2c-4078-93db-1edbf8c2d05b.html	
+++ b/output/Basic Navigation with Current Page/3/f8a4851c-2e2c-4078-93db-1edbf8c2d05b.html	
@@ -1,21 +1,12 @@
-<ul>
-  <li><a href="/">Home</a></li>
-  <li><a href="/products" class="current">Products</a>
-    <ul>
-      <li><a href="/products/frobs">Frobs</a></li>
-      <li><a href="/products/thrips">Thrips</a></li>
-    </ul>
-  </li>
-  <li><a href="/services">Services</a></li>
-</ul>
-<style>
-  .current {
-    font-weight: bold;
-  }
-  ul ul {
-    display: none;
-  }
-  li:hover > ul {
-    display: block;
-  }
-</style>
\ No newline at end of file
+<nav>
+  <ul>
+    <li><a href="/">Home</a></li>
+    <li><a href="/products" class="current" aria-current="page">Products</a>
+      <ul>
+        <li><a href="/products/frobs">Frobs</a></li>
+        <li><a href="/products/thrips">Thrips</a></li>
+      </ul>
+    </li>
+    <li><a href="/services">Services</a></li>
+  </ul>
+</nav>
\ No newline at end of file
